### PR TITLE
[TT-10117] Update documentation for 5.0 LTS

### PR DESCRIPTION
**Pull request by buger**
Triggered by: lghiur

Included:

Tyk Gateway: true
Tyk Dashboard: true
Tyk MDCB false
Tyk Pump false

Intended for: 5.0 LTS

Note: Docs updates s for 5.0.6

JIRA: https://tyktech.atlassian.net/browse/TT-10117

### CODE REVIEW

File: tyk-docs/assets/others/dashboard-swagger.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   title: Tyk Dashboard API
-  version: 5.0.0
+  version: 4.3.0
```
Comment (dcs3spp) [inline tyk-docs/assets/others/dashboard-swagger.yml]: Should the version be 4.3?

File: tyk-docs/assets/others/gateway-swagger.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   title: Tyk Gateway API
-  version: 5.0.0
+  version: 4.3.0
```
Comment (dcs3spp) [inline tyk-docs/assets/others/gateway-swagger.yml]: Should the version for 5.0 LTS be 5.0.x?

File: tyk-docs/assets/others/dashboard-swagger.yml (hunk 1 above)
Comment (titpetric) [inline tyk-docs/assets/others/dashboard-swagger.yml]:
```suggestion
  version: 5.0.0
```

Not necessary change (we don't update this version on source)...

File: tyk-docs/assets/others/gateway-swagger.yml (hunk 2 above)
Comment (titpetric) [inline tyk-docs/assets/others/gateway-swagger.yml]:
```suggestion
  version: 5.0.0
```

File: tyk-docs/content/shared/x-tyk-gateway.md [hunk 3]
```diff
@@ -193,9 +193,18 @@ Tyk classic API definition: `service_discovery.use_target_list`.
 **Field: `cacheTimeout` (`int`)**
 CacheTimeout is the timeout of a cache value when a new data is loaded from a discovery service.
 Setting it too low will cause Tyk to call the SD service too often, setting it too high could mean that failures are not recovered from quickly enough.
+Deprecated: The field is deprecated, usage needs to be updated to configure caching.
```
Comment (dcs3spp) [inline tyk-docs/content/shared/x-tyk-gateway.md:196]: What field should be used instead to configure caching timeout..is it the cache field at line 200?

```summary
Deprecated: The field is deprecated. Please use *service_discovery.cache_timeout* to configure caching timeout.
```

File: tyk-docs/assets/others/dashboard-swagger.yml (hunk 1 above)
Comment (dcs3spp) [inline tyk-docs/assets/others/dashboard-swagger.yml]: Should the version for 5.0 LTS be 5.0.x?

**Review by dcs3spp — CHANGES_REQUESTED** [review]
lgtm, only version update from 4.3.0

File: tyk-docs/assets/others/dashboard-swagger.yml (hunk 1 above)
Comment (lghiur) [inline tyk-docs/assets/others/dashboard-swagger.yml]:
```suggestion
  version: 5.0.6
```

**Review by lghiur — APPROVED** [review]

File: tyk-docs/assets/others/gateway-swagger.yml (hunk 2 above)
Comment (lghiur) [inline tyk-docs/assets/others/gateway-swagger.yml]:
```suggestion
  version: 5.0.6
```

**Review by lghiur — APPROVED** [review]

**Review by dcs3spp — APPROVED** [review]

**Pull request merged**